package.json

```diff
@@ -29,7 +29,8 @@
     "ci:lint": "npm run lint && npm run security",
     "ci:test": "npm run test -- --runInBand",
     "ci:coverage": "npm run test:coverage -- --runInBand",
-    "defaults": "webpack-defaults"
+    "defaults": "webpack-defaults",
+    "webpack-defaults": "webpack-defaults"
```

Comment:
```diff
-   "webpack-defaults": "webpack-defaults"
```

Reply:
Hmm how the hell did that end up in there? lol

Reply:
It's a left over from the `webpack-defaults` `v1.6.0...2.0.0` upgrade one needs to be aware of, unrelated to this PR entirely. I just saw/catched it here :)

Reply:
Do I need to make a change for this? Or is PR good as is?